dev/io.openliberty.org.testcontainers/.gitignore

```diff
@@ -0,0 +1 @@
+cache
```

Comment:
I think this change is wrong. The testcontainers cache is auto-updated by the build and is supposed to be checked in.